.stylelintrc.json

```diff
@@ -4,8 +4,6 @@
     "stylelint-use-logical-spec"
   ],
   "rules": {
-    "indentation": 2,
-
```

Comment:
This seems suspicious? Why is it dropped?

Reply:
https://stylelint.io/migration-guide/to-15 and https://stylelint.io/user-guide/rules/indentation/ just say that it's deprecated, but not why :man_shrugging:

Reply:
Ugh, well as the default is 2 anyway. LGTM.

Reply:
Oh, that could be? They want to stop supporting anything else than 2?